integtest/.rubocop.yml

```diff
@@ -0,0 +1,6 @@
+inherit_from: ../.rubocop.yml
+
+
+Metrics/ClassLength:
+  Exclude:
+    - spec/helper/dest.rb
```

Comment:
There are a couple of TODOs that I haven't gotten around to for a few months that will make this pass once I do them. I didn't want to do them as part of this change. I'll get them in a follow up and remove this.